dedibox-network/domains/how-to/change-ownership.mdx

```diff
@@ -5,24 +5,25 @@ meta:
 content:
   h1: How to change the ownership of a domain name
   paragraph: This page explains how to change the ownership of a domain name
-tags: dedibox dedibox-network domain-name domain 
 dates:
-  validation: 2023-09-19
+  validation: 2023-12-14
   posted: 2021-05-26
-categories:
-  - dedibox-network
 ---
 
-You must always keep the contact details relating to the owner of your domain name accurate and up-to-date. This can be done by [following the instructions here](/dedibox-network/domains/how-to/file-contact-info). However, if you wish to change details pertaining to the identity of the owner (full name(s) of a natural person or corporate name for a legal entity) this must be done by the change of ownership process described below.
+You must always keep the contact details relating to the owner of your domain name accurate and up to date. This can be done by [following the instructions here](/dedibox-network/domains/how-to/file-contact-info). However, if you wish to change details pertaining to the identity of the owner (full name(s) of a natural person or corporate name for a legal entity) this must be done by the change of ownership process described below.
```

Comment:
```suggestion
You must always keep the contact details relating to the owner of your domain name accurate and up to date. This can be done by [following the instructions here](/dedibox-network/domains/how-to/file-contact-info). However, if you wish to change details pertaining to the identity of the owner (full name(s)) of a natural person or corporate name for a legal entity) this must be done by the change of ownership process described below.
```

Reply:
Already a closing bracket after "legal entity"